docs/source/platform_technical_documentation/ade_custom_extensions/jobsui.ipynb

```diff
@@ -20,7 +20,7 @@
     "1. Open your workspace\n",
```

Comment:
<p>Note: do not see this option on workspace launcher tab (MAAP Basic Stable workspace)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
This documentation applies to the DIT and UAT venues now, but will also apply to the OPS venue upon the upcoming deployment. Judging by comments in this thread, it looks like the OPS venue was used.